api/catalog/api/utils/check_dead_links.py

```diff
@@ -119,7 +119,14 @@ def validate_images(
     for idx, _ in enumerate(cached_statuses):
         del_idx = len(cached_statuses) - idx - 1
         status = cached_statuses[del_idx]
-        if status == 429 or status == 403:
+        # thingiverse treated as failure despite the suspect status code
```

Comment:
We could extract a function like `is_request_rate_limited_or_blocked(status, provider)` to make this part of code easier to read - but this is not a blocking comment.